lms/djangoapps/teams/static/teams/js/collections/topic.js

```diff
@@ -1,4 +1,4 @@
-;(function (define) {
+(function(define) {
```

Comment:
Is there a reason that the line would have intentionally started with a ";" ? (I'm not very familiar with js code).